src/bpftrace.cpp

```diff
@@ -1304,7 +1305,7 @@ int BPFtrace::print_lhist(const std::vector<uint64_t> &values, int min, int max,
     std::ostringstream header;
     if (i == 0) {
       header << "(..., " << lhist_index_label(min) << ")";
-    } else if (i == (buckets + 1)) {
+    } else if (i == (static_cast<int>(buckets) + 1)) {
```

Comment:
Does this avoid overflow or does it hide it?

According to the c++17 standard section 7.8/3:
```
If the destination type is signed, the value is unchanged if it can be represented in the destination type;
otherwise, the value is implementation-defined
```

IIUC, casting to `long int` should be more correct.

Reply:
AFAICS this is not a problem...

I can't see a scenario where the number of `buckets` can overflow an `int` (2,147,483,647). Usually the value is very low.

I'll update this PR because I just realized that I don't need to change 3 lines, just 1.

Reply:
Yeah I'm just being pedantic here. I agree it probably isn't a problem.